runtime/stout/src/lib.rs

```diff
@@ -779,18 +779,6 @@ impl_runtime_apis! {
 		}
 	}
 
-	#[cfg(feature = "try-runtime")]
```

Comment:
if you are removing conditional compilation from stout, it probably makes sense remove `try-runtime` feature from cargo.toml of stout.

Reply:
Done.